### ISSUE TITLE
tests are bound to a remote IP instead of the current server

### ISSUE THREAD

**Issue by jason-lopez-n** [macortesn/prueba-valiu#2]
https://github.com/macortesn/prueba-valiu/blob/4a8fd51a31402f3c2777a5bfac843c9543995418/test/app_test.js#L8

API test should be bound to the current server (express app in [index.js](https://github.com/macortesn/prueba-valiu/blob/master/src/index.js)) instead of using a remote IP.

**Comment by macortesn** [comment]
Yes sir, is that the tests were the last thing I did, then use the API IP deployed in the cloud.

**Comment by S0c5** [comment]
## Feature request

Please migrate the test to use the current instance of express server and not the remote server.
